cms/djangoapps/contentstore/views/transcripts_ajax.py

```diff
@@ -87,6 +91,45 @@ def link_video_to_component(video_component, user):
     return edx_video_id
 
 
+def save_video_transcript_in_learning_core(
+    usage_key,
+    input_format,
+    transcript_content,
+    language_code
+):
+    """
+    Saves a video transcript to the learning core
```

Comment:
Might be worth adding a comment/docstring that SJSON is an edx-specific format that we're trying to move away from, so all new (Learning Core) stuff should only use the standard `.srt` format for subtitles.

Reply:
Added in as comment https://github.com/openedx/edx-platform/pull/36058/commits/6903be2b674410d06f07572f7acc6c37b3f23482